tst/laguna04.tst

```diff
@@ -698,9 +698,7 @@ true
 
 # doc/funct.xml:1937-1952
```

Comment:
@wilfwilson did you update the .tst file manually or it's autogenerated? I am suspicious because the location of this example in the xml file did not change.

Reply:
Thanks for spotting that @alex-konovalov - I had done it manually, but now I have done it properly with the autogeneration.

Reply:
Perfect @wilfwilson